packages/devtools_app/lib/src/shared/dialogs.dart

```diff
@@ -5,13 +5,21 @@
 import 'package:flutter/material.dart';
 import 'package:pointer_interceptor/pointer_interceptor.dart';
 
+import '../ui/label.dart';
 import 'common_widgets.dart';
 import 'theme.dart';
 
 const dialogDefaultContext = 'dialog';
 
-Text dialogTitleText(ThemeData theme, String text) {
-  return Text(text, style: theme.textTheme.titleLarge);
+class DialogTitleText extends StatelessWidget {
+  const DialogTitleText({super.key, required this.text, required this.theme});
+
+  final ThemeData theme;
```

Comment:
we shouldn't be passing the theme in. We can access the theme in the build method directly.